packages/utils/telemetry-utils/src/test/types/validateTelemetryUtilsPrevious.generated.ts

```diff
@@ -440,28 +440,28 @@ use_old_InterfaceDeclaration_ITelemetryLoggerExt(
  * If this test starts failing, it indicates a change that is not forward compatible.
  * To acknowledge the breaking change, add the following to package.json under
  * typeValidation.broken:
- * "InterfaceDeclaration_ITelemetryLoggerPropertyBag": {"forwardCompat": false}
+ * "RemovedInterfaceDeclaration_ITelemetryLoggerPropertyBag": {"forwardCompat": false}
```

Comment:
IIRC all these are because these interfaces are now @alpha and thus type tests mostly validate public things? This might have been discussed and agreed upon already, but I still find it a bit weird, since we still want to be careful about not breaking alpha APIs inadvertently.

Reply:
Ah, reverted in the latest version, then I'm much less concerned :).